src/dev/build/tasks/os_packages/docker_generator/resources/bin/kibana-docker

```diff
@@ -82,6 +82,14 @@ kibana_vars=(
     vega.enableExternalUrls
     xpack.apm.enabled
     xpack.apm.ui.enabled
+    xpack.apm.ui.maxTraceItems
+    apm_oss.apmAgentConfigurationIndex
+    apm_oss.indexPattern
+    apm_oss.errorIndices
+    apm_oss.onboardingIndices
+    apm_oss.spanIndices
+    apm_oss.transactionIndices
+    apm_oss.metricsIndices
```

Comment:
@sqren i added these other configs to the docker template since i noticed they were missing. maybe they weren't copied over from the old docker template

Reply:
Awesome! Good find 👍